src/Utils/Validate.js

```diff
@@ -4,8 +4,11 @@ const Required = (value) => {
   return 'Required!';
 };
 
-const RequiredInput = () => {
-  return false;
+const isURLValid = (value) => {
+  const regex = new RegExp('^$|([Hh][Tt][Tt][Pp]|[Ff][Tt][Pp])([Ss])?://.+$');
+  const isTrue = regex.test(value);
+  if (isTrue) return undefined;
```

Comment:
personally for me `isTrue` doesn't tell much what is the `true`, maybe it's more readable like this:
```
return REGEXP_URL.test(value)
  ? undefined
  : 'Invalid URL';
```

Reply:
thanks @aliaksei-chumakou but I prefer this format. let me update the rest of the code. thanks for suggestion.